### PR TITLE
Tvs rj/ux peerprogramming

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
Feedback and updates made from pair programming session with Rasmus.

The main focus is to bring consistency to the padding and font sizes / weights across the tooltips and headers.

### CODE REVIEW

File: web/src/components/CarbonIntensitySquare.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ function CarbonIntensitySquare({
           }}
           className="mx-auto flex h-[65px] w-[65px] flex-col items-center justify-center rounded-2xl"
         >
-          <p className="select-none" data-test-id="co2-square-value">
+          <p className="select-none text-[1rem]" data-test-id="co2-square-value">
```
Comment (madsnedergaard) [inline web/src/components/CarbonIntensitySquare.tsx:54]: Why add this, isn't it 1 rem by default? 🤔

File: web/src/features/exchanges/ExchangeTooltip.tsx [hunk 2]
```diff
@@ -14,8 +14,8 @@ export function CarbonIntensity({ intensity }: { intensity?: number }) {
         className="mr-1 h-3 w-3"
         style={{ backgroundColor: co2ColorScale(intensity ?? 0) }}
       />
-      <b className="flex items-center">{Math.round(intensity ?? 0) || '?'}</b>
-      <p className="pl-0.5"> gCO₂eq/kWh</p>
+      <b className="flex items-center font-bold">{Math.round(intensity ?? 0) || '?'}</b>
+      <p className="pl-0.5 font-light"> gCO₂eq/kWh</p>
```
Comment (madsnedergaard) [inline web/src/features/exchanges/ExchangeTooltip.tsx:18]: We only load Inter in 400-700, so this doesn't change anything AFAIK :)

File: web/src/components/CircularGauge.tsx [hunk 3]
```diff
@@ -42,7 +42,7 @@ export function CircularGauge({ ratio, name, tooltipContent }: CircularGaugeProp
               strokeWidth={0}
             >
               <Label
-                className="select-none fill-gray-900 font-bold dark:fill-gray-300"
+                className="select-none fill-gray-900 text-[1rem] font-bold dark:fill-gray-300"
```
Comment (madsnedergaard) [inline web/src/components/CircularGauge.tsx:45]: Why add this, isn't it 1 rem by default? 🤔

**Review by madsnedergaard — APPROVED** [review]
Nice tweaks! I have a few minor nitpicky comments, otherwise it looks great 🥳

File: web/src/features/exchanges/ExchangeTooltip.tsx (hunk 2 above)
Comment (tonypls) [reply web/src/features/exchanges/ExchangeTooltip.tsx]: removed! super hard to tell minor differences with tooltips, hmm if only there was a great tool like story book for this kind of stuff 🤷

**Pull request merged**